chart/templates/_helpers.yaml

```
@@ -664,6 +664,12 @@ server_tls_key_file = /etc/pgbouncer/server.key
   {{- include "_serviceAccountName" (merge (dict "key" "triggerer") .) -}}
 {{- end }}
 
+{{/* PR 48032 renames triggerer.default_capacity to capacity, and falls back to a default value */}}
```

```suggestion
{{/* Determine trigger capacity, taking Airflow 2 and 3 config option differences into account*/}}
```

Generally, we try not to place pr numbers around - we can always get at that by doing a blame - so it's better to just explain what this does instead.